web/sites/default/config/webform.webform.apply_tsup.yml

```diff
@@ -2419,7 +2512,48 @@ elements: |-
+      {% set source_url = webform_token('[webform_submission:source-url:path]', webform_submission) %}
+      {% if data.permit_type == "sidewalk_repair" %}
+      <p>Sidewalk repair notifications cannot include any street or travel lane closures. For these closures, please submit a separate permit application by using the button below.</p>
+      {% endif %}
+      {% if data.wizard_permit_need != "general" and source_url %}
+      {% set app_url = source_url ~ "?" ~ ({
+          assessment_prefilled: 1,
+          wizard_permit_need: data.permit_type == "sidewalk_repair" ? "traffic" : data.wizard_permit_need,
+          wizard_existing_permit_situation: data.wizard_existing_permit_situation,
+          general_subpage: data.general_subpage,
+          contact_full_legal_name: data.contact_full_legal_name|url_encode,
+          contact_company: data.contact_company|url_encode,
+          contact_phone: data.contact_phone|url_encode,
+          contact_email: data.contact_email|url_encode,
+          contact_applicant_type: data.permit_type == "sidewalk_repair" ? "" : data.contact_applicant_type|url_encode,
+          contact_city_work_type: data.contact_city_work_type|url_encode,
+          contact_fun_company: data.contact_fun_company|url_encode,
+          contact_bureau: data.contact_bureau|url_encode,
+          complex_permit_billing_info: data.complex_permit_billing_info|url_encode,
+          complex_permit_bureau_contact: data.complex_permit_bureau_contact|url_encode,
+          complex_permit_job_number: data.complex_permit_job_number|url_encode,
+          complex_permit_self_doing_work: data.complex_permit_self_doing_work|url_encode,
+          contact_subcontractor_company: data.contact_subcontractor_company|url_encode,
+          contact_subcontractor_name: data.contact_subcontractor_name|url_encode,
+          contact_subcontractor_phone: data.contact_subcontractor_phone|url_encode,
+          contact_subcontractor_email: data.contact_subcontractor_email|url_encode,
+        }|filter((v) => v is not empty)|map((v, k) => k ~ "=" ~ v)|join("&")) %}
+      <p><a class="btn btn-primary" href="{{ app_url }}{% if data.permit_type == "fun" or data.permit_type == "sidewalk_repair" %}&contact_prefilled=1{% endif %}">
```

Comment:
I see you've added target=_blank to several other links in the form. Does this link need that as well?